android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -265,12 +265,8 @@ constructor(
     questionnaire: Questionnaire,
     bundle: Bundle?
   ) {
-    if (!questionnaireConfig.resourceIdentifier.isNullOrEmpty() ||
```

Comment:
@dubdabasoduba , @maimoonak 
this check was blocking to call extractCarePlan while registering new Family

cc: @f-odhiambo , @ageryck